test/mocks.js

```diff
@@ -141,6 +164,8 @@ function mockDB (data, errors) {
         wrapWrapKb: data.wrapWrapKb
       })
     }),
+    createEmail: sinon.spy(() => { return P.resolve({})}),
+    deleteEmail: sinon.spy(() => { return P.resolve({})}),
```

Comment:
Could we get by with the default implementation of these from `mockObject` (it returns `P.resolve()`)? If not, does it break any tests if we update that default implementation to return `P.resolve({})` so we can use it here?

Also, no biggie but we could ditch the explicit `return` from these if we lost the `{}`.